ios/Capacitor/Capacitor/Plugins/Camera.swift

```diff
@@ -123,41 +111,40 @@ public class CAPCameraPlugin : CAPPlugin, UIImagePickerControllerDelegate, UINav
       call.error("Camera not available while running in Simulator")
       return
     }
-    
-    guard let imagePicker = self.imagePicker else {
-      call.error("Internal error, please file an issue")
-      return
+
+    let presentationStyle = call.getString("presentationStyle")
+    if presentationStyle != nil && presentationStyle == "popover" {
+      self.configurePicker()
+    } else {
+        self.imagePicker!.modalPresentationStyle = .fullScreen
     }
+
+    self.imagePicker!.sourceType = .camera
     
-    imagePicker.sourceType = .camera
-    
-    self.bridge.viewController.present(imagePicker, animated: true, completion: nil)
+    self.bridge.viewController.present(self.imagePicker!, animated: true, completion: nil)
   }
-  
+
   func showPhotos(_ call: CAPPluginCall) {
     let photoAuthorizationStatus = PHPhotoLibrary.authorizationStatus()
     if photoAuthorizationStatus == .restricted || photoAuthorizationStatus == .denied {
       call.error("User denied access to photos")
       return
     }
     
+    self.configurePicker()
+    
     self.imagePicker!.sourceType = .photoLibrary
     self.imagePicker!.allowsEditing = settings.allowEditing
     
     self.bridge.viewController.present(self.imagePicker!, animated: true, completion: nil)
   }
-  
-  func getSourceType(_ source: CameraSource) -> UIImagePickerControllerSourceType {
```

Comment:
How are we doing source type now?

Reply:
showCamera sets it to .camera 
https://github.com/ionic-team/capacitor/blob/master/ios/Capacitor/Capacitor/Plugins/Camera.swift#L132
showPhotos to .photoLibrary
https://github.com/ionic-team/capacitor/blob/master/ios/Capacitor/Capacitor/Plugins/Camera.swift#L144

So that function was not really doing anything

Reply:
Awesome